automation/Dockerfile-tests

```diff
@@ -1,4 +1,4 @@
-FROM iflavoursbv/sbt-openjdk-8-alpine
+FROM bigtruedata/sbt
```

Comment:
using `bigtruedata/sbt` due to some grpc error similar to what I ran into in sam when using firestore client